src/main/java/co/omise/models/ModelTypeResolver.java

```diff
@@ -6,30 +6,40 @@
 import com.fasterxml.jackson.databind.DatabindContext;
 import com.fasterxml.jackson.databind.JavaType;
 import com.fasterxml.jackson.databind.jsontype.impl.TypeIdResolverBase;
-import com.google.common.collect.ImmutableBiMap;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 class ModelTypeResolver extends TypeIdResolverBase {
-    static final ImmutableBiMap<String, Class> KNOWN_TYPES = new ImmutableBiMap.Builder<String, Class>()
-            .put("account", Account.class)
-            .put("balance", Balance.class)
-            .put("bank_account", BankAccount.class)
-            .put("card", Card.class)
-            .put("charge", Charge.class)
-            .put("customer", Customer.class)
-            .put("dispute", Dispute.class)
-            .put("event", Event.class)
-            .put("link", Link.class)
-            .put("refund", Refund.class)
-            .put("recipient", Recipient.class)
-            .put("schedule", Schedule.class)
-            .put("occurrence", Occurrence.class)
-            .put("token", Token.class)
-            .put("transaction", Transaction.class)
-            .put("transfer", Transfer.class)
-            .put("source", Source.class)
-            .put("receipt", Receipt.class)
-            .put("forex", Forex.class)
-            .build();
+    private static Map<String, Class> types;
+
+    static Map<String, Class> getKnownTypes() {
+        if (types == null) {
```

Comment:
We could failfast here and saves an indentation.

Reply:
I don't think we want to fail here, we just want to check if the array is already populated or not.